apps/rush-lib/src/cli/actions/PublishAction.ts

```diff
@@ -263,6 +272,8 @@ export default class PublishAction extends BaseRushAction {
 
       if (this._npmTag.value) {
         args.push(`--tag`, this._npmTag.value);
+      } else if (this._hotfixTagOverride) {
+        args.push(`--tag`, this._hotfixTagOverride);
```

Comment:
This is not needed.

Reply:
If a tag is not passed to npm publish, the 'latest' tag is automatically used and we don't want the hotfix version to replace the latest on the package manager.